server/runtime_javascript.go

```diff
@@ -1809,25 +1812,25 @@ func (rp *RuntimeProviderJS) MatchmakerMatched(ctx context.Context, entries []*M
 	entriesSlice := make([]interface{}, 0, len(entries))
 	for _, e := range entries {
 		presenceObj := r.vm.NewObject()
-		presenceObj.Set("userId", e.Presence.UserId)
-		presenceObj.Set("sessionId", e.Presence.SessionId)
-		presenceObj.Set("username", e.Presence.Username)
-		presenceObj.Set("node", e.Presence.Node)
+		_ = presenceObj.Set("userId", e.Presence.UserId)
```

Comment:
I guess it's more of a personal preference here, I'd personally always prefer to explicitly show that I willingly ignore the returned errors and not just forgot to handle it. I agree about the `defer` though, I'll take a look if it's possible to configure the linter to avoid having to annotate those.

Reply:
Added a rule to ignore the `cleanup` function. By default it actually ignores quite a lot of stdlib functions in defer so this was the only custom edge case.